crates/zk-por-core/src/util.rs

```diff
@@ -0,0 +1,44 @@
+pub fn get_recursive_hash_nums(num_of_batches: usize, hyper_tree_leaf_size: usize) -> usize {
```

Comment:
add some comments for this function, as I cannot figure out what this function does.